tests/virt/cluster/migration_and_maintenance/rhel9_rhel10_cluster/test_live_migration.py

```
@@ -50,3 +130,9 @@ def test_vm_migrates_from_rhcos9_to_rhcos10_node():
         - Live migration completes successfully and the VM is running on an RHCOS 10 worker node
         - The VM did not restart during migration
     """
+    set_vm_affinity(vm=dual_stream_migration_vm, affinity=RHCOS10_AFFINITY)
+    migrate_vm_and_verify(vm=dual_stream_migration_vm, client=admin_client, check_ssh_connectivity=True)
+    assert (
+        RHCOS9_WORKER_LABEL not in dual_stream_migration_vm.vmi.get_node(privileged_client=admin_client).labels.keys()
```

Ruff (0.15.20)

[warning] Use `key not in dict` instead of `key not in dict.keys()`

Remove `.keys()`

(SIM118)